juego.py

```diff
@@ -133,28 +133,22 @@ class Game():
 
     def __init__(self, activity):
         self.activity = activity
-        self.locale = locale.getdefaultlocale()[0]
-        if self.locale not in ("en_US", "es_ES"):
-            self.locale = "en_US"
-        self.menu_texts = {
-            "en_US": ["PLAY", "LEVEL", "QUIT"],
-            "es_US": ["JUGAR", "NIVEL", "SALIR"],
-        }[self.locale]
-        self.difficuly_texts = {
-            "en_US": ["easy", "medium", "hard"],
-            "es_US": ["facil", "medio", "dificil"],
-        }[self.locale]
-        self.status_text = {
-            "en_US": ["Score : ", "Highest Score : ", "Timer : "],
-            "es_US": ["Puntaje : ", "Puntaje Mas Alto : ", "Temporizador : "],
-        }[self.locale]
-        self.game_over_text = {
-            "en_US": ["PLAY AGAIN", "QUIT", "GAME OVER!!", "Hurray! you won :)","Ay! you lost :("],
-            "es_US": ["jUEGA DE NUEVO", "SALIR", "JUEGO TERMINADO!!", "Hurra! ganaste :)","Ay! perdiste :("],
-        }[self.locale]
+        self.locale = locale.getdefaultlocale()[0][:2]
+        if self.locale not in ("en", "es"):
+            self.locale = "en"
+        self.translations = {
+            "en": ["PLAY", "LEVEL", "QUIT", "easy", "medium", "hard", "Score : ", "Highest Score : ", "Timer : ","PLAY AGAIN", "GAME OVER!!", "Hurray! you won :)", "Ay! you lost :("],
+            "es" : ["JUGAR", "NIVEL", "SALIR", "facil", "medio", "dificil", "Puntaje : ", "Puntaje Mas Alto : ", "Temporizador : ", "jUEGA DE NUEVO", "JUEGO TERMINADO!!", "Hurra! ganaste :)", "Ay! perdiste :("],
+        }
+        
```

Comment:
These are really long lines, also this can be part of `get_translated_text` and the function shouldn't be part of the activity class as the only thing it does is to return text and nothing directly related to the game functionality.